src/httpResponses.js

```diff
@@ -0,0 +1,59 @@
+const httpResponses = [
+  {
+    "code": 200,
```

Comment:
We can remove this 200 code, the response will come back ok if the code is below 300.

Reply:
You're right. I just pushed an update for this.